web-registry/src/pages/Media.tsx

```diff
@@ -38,8 +41,20 @@ const Media: React.FC = () => {
     return Promise.resolve();
   };
 
+  function navigateNext(): void {
+    // TODO: replace 'review' with path name once
+    // https://github.com/regen-network/regen-registry/issues/899 is merged
```

Comment:
@blushi wanted to check here - will we have two metadata routes? I see a non-VCS metadata form in the mockups, and wasn't sure whether to route there or `/review` (as i'm currently doing)

Reply:
https://github.com/regen-network/regen-registry/issues/899 is not related to this project creation flow at all though, but to the credit batch creation flow

Reply:
this issue https://github.com/regen-network/regen-registry/issues/447 should be referenced instead